### DISCUSSION TITLE
cron backup

### DISCUSSION THREAD

**Discussion by dmbrtlt** [totumonline/totum-ru-issues-and-discussions#607] [ОШБК!] [answered]
При попытке выполнить крон бэкапа (нажав на кнопку), выскакивает ошибка:
This option works only in PRO.
action поле do_it_now таблицы crons id 6
Но установлена именно PRO версия
<img width="1222" alt="Снимок экрана 2024-10-13 в 12 03 53" src="https://github.com/user-attachments/assets/2834cdd4-b6f5-4ea4-9890-1eae96338374">

**Comment by ttm-alexeypolunin** [comment]
Что у вас в коде, который вы пытаетесь выполнить?

и какая версия?

**Reply by ttm-alexeypolunin** [reply]
Проверили. У нас не повторяется на версии 5.13.60.0-8.2

Но если у вас все-таки не переключено на PRO, то execSSH, который используется в предположительно вашем коде, действительно доступен только в PRO:

```
=: while(action: $a1; action: $a2)

a1: execSSH(ssh: "cd ~/totum-mit && bin/totum schema-backup --schema='totum' --gz '%schema%-%Y%-%m%-%d%.sql'")

a2: execSSH(ssh: str`"cd ~/totum-mit && rm totum-" + $dm + ".sql.gz"`)

        dm: dateAdd(date: $#nd; days: -7; format: "Y-m-d")
```

**Reply by dmbrtlt** [reply]
Где посмотреть версию? Сейчас предварительно обновился (ошибка осталась), 
Как принудительно переключиться на PRO? (хотя устанавливал именно ее)

**Reply by dmbrtlt** [reply]
вот еще посмотрел ошибку при установке
TASK [Perform SSH to GitHub] ***************************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "ssh -T git@github.com", "delta": "0:00:00.232311", "end": "2024-10-12 22:45:45.893824", "failed_when_result": true, "msg": "non-zero return code", "rc": 255, "start": "2024-10-12 22:45:45.661513", "stdout": "ssh: Could not resolve hostname github.com: Temporary failure in name resolution", "stdout_lines": ["ssh: Could not resolve hostname github.com: Temporary failure in name resolution"]}

**Reply by dmbrtlt** [reply, marked as answer]
Переустановил, из за неверного ключа ssh и не работало, сейчас все ок
